modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/LiferayOSGiDefaultsPlugin.java

```diff
@@ -1297,11 +1295,11 @@ public String doCall(String content, File file) {
 						return content;
 					}
 
-					File modulesDir = new File(portalRootDir, "modules");
-
-					String relativePath = FileUtil.relativize(file, modulesDir);
+					File markerFile = GradleUtil.getRootDir(
```

Comment:
The `getRootDir` method returns a directory, so the variable should be called `dir`, `markerDir`, or something like that.